pkg/ocm/helpers.go

```diff
@@ -526,3 +528,31 @@ func CheckSupportedVersion(clusterVersion string, operatorVersion string) (bool,
 	//Cluster version is greater than or equal to operator version
 	return v1.GreaterThanOrEqual(v2), nil
 }
+
+func SaveDocument(doc string, filename string) error {
+	file, err := os.Create(filename)
+	if err != nil {
+		return err
+	}
+	defer file.Close()
+
+	_, err = file.WriteString(doc)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func (c *Client) AddOperatorRole(cluster *cmv1.Cluster, operator aws.Operator) error {
```

Comment:
@pvasant we can remove this function now since we're going to add the operator role in CS create upgrade policy.

Reply:
removed